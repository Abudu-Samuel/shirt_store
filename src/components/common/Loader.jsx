import React from 'react';
import Loader from "react-loader-spinner";

export default () => (
  <div className="loader">
    <Loader
      type="CradleLoader"
      color="#f7436b"
      height="100"
      width="100"
      className="loader"
    />
  </div>
)