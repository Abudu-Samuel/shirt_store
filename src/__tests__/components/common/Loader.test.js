import React from "react";
import {
  shallow
} from "enzyme";
import toJSON from "enzyme-to-json";
import Loader from "../../../components/common/Loader";

describe("<Loader />", () => {
  it("renders without crashing", () => {
    const wrapper = shallow( < Loader / > );
    expect(toJSON(wrapper)).toMatchSnapshot();
  });
});