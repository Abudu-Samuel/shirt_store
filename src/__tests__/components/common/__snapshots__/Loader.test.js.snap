// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Loader /> renders without crashing 1`] = `
<div
  className="loader"
>
  <Loader
    className="loader"
    color="#f7436b"
    height="100"
    type="CradleLoader"
    width="100"
  />
</div>
`;
